config/middleware.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const helmet = require('helmet');
+const morgan = require('morgan');
+const cors = require('cors');
+
+module.exports = server => {
+  server.use(express.json());
+  server.use(helmet());
```

Comment:
Generally you want helmet at the top of your stack for security purposes. (It does something w/ protecting the headers from being seen which could expose your stack)